artemis-junit/src/test/java/org/apache/activemq/artemis/junit/ActiveMQConsumerResourceTest.java

```diff
@@ -50,7 +50,7 @@
    ActiveMQConsumerResource consumer = new ActiveMQConsumerResource(server.getVmURL(), TEST_QUEUE);
 
    @Rule
-   public RuleChain ruleChain = RuleChain.outerRule(new ThreadLeakCheckRule()).outerRule(server).around(consumer);
+   public RuleChain ruleChain = RuleChain.outerRule(new ThreadLeakCheckRule()).around(server).around(consumer);
```

Comment:
Trying this on multiple machines, outside of debug mode it passes on my rhel vm, but not on bare metal laptop. When running under debugger, it passes everywhere. I have the same problem with other tests that use this rulechain construct, such as `MultipleEmbeddedActiveMQResourcesTest`.

```
*********************************************************************************
LEAKING THREADS
=============================================================================
Thread Thread[Attach Listener,9,system] is still alive with the following stackTrace:
*********************************************************************************
[main] 09:39:58,447 INFO  [org.apache.activemq.artemis.junit.ThreadLeakCheckRule] #test forceGC
[main] 09:39:58,500 INFO  [org.apache.activemq.artemis.junit.ThreadLeakCheckRule] #test forceGC Done 
*********************************************************************************
```